config/bootstrap.php

```diff
@@ -61,11 +39,24 @@
 use Cake\Database\Type;
 use Cake\Datasource\ConnectionManager;
 use Cake\Error\ErrorHandler;
+use Cake\Http\ServerRequest;
 use Cake\Log\Log;
 use Cake\Mailer\Email;
-use Cake\Network\Request;
 use Cake\Utility\Security;
 
+/**
+ * Uncomment block of code below if you want to `.env` file during development.
```

Comment:
typo? "if you want to X `.env`"... X = "use"?

Reply:
Yeah, done. There was a typo also in cakephp/app. I have subimtted a PR https://github.com/cakephp/app/pull/551